src/transformers/configuration_utils.py

```diff
@@ -55,6 +55,7 @@ class PretrainedConfig(object):
 
     def __init__(self, **kwargs):
         # Attributes with defaults
+        self.return_tuple = kwargs.pop("return_tuple", False)
```

Comment:
Does this need to be decided at `__init__` in order to support torchscript? Can it be decided as a kwarg to forward?

Reply:
Both are supported, like `output_attentions` or `output_hidden_states`.

Reply:
I'm wondering if we need the `self.return_tuple` variable here at all. As I understand your approach is fully backwards compatible and we only need to switch back to classic tuples if we use `torchscript` => so would having only the property `use_return_tuple` => `return self.torchscript` not be enough?

Reply:
Is there ever a case where a user would not want the "nice version of named tuples"?

Reply:
Note that `self.torchscript` implies other changes (such as untying the weights) so it can'be used to replace `self.return_tuple`. 
Also note that the `ModelOuput` has some of the attributes of a tuple but not all (for instance you can't concatenate them).

I don't think adding an optional flag hurts. Yes, it's probably never going to be used by 99% of the users since the rest is fully backward compatible, but for the 1% that would need it? I don't see the harm in providing a way to revert to the old API since it only takes that one line of code (and the `use_return_tuple` property).